components/Notifications/SubscribedAuthors.js

```diff
@@ -148,7 +144,7 @@ const SubscribedAuthors = ({
                 >
                   <div {...styles.author}>
                     <Link href={`/~${author.userDetails.slug}`}>
-                      <A {...styles.userLink}>{author.object.name}</A>
+                      <A {...styles.authorLink}>{author.object.name}</A>
```

Comment:
A few inputs here:
- [avoid custom elements](https://styleguide.republik.ch/dev/colors#avoid-custom-elements)—this rule has been formulated in the context of color schema—but applied generally
- Cursor pointer is missing because the href is not passed to the a tag: add `passHref` to `<Link>`
- IMO we should use `Editorial.A`—too many green links are annoying :D

Reply:
I will replace the custom css-rule and use the passHref attribute. 👍🏻